From 126920ea133a7601ea71c48a6dd1948abd7e3c0c Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 24 Sep 2015 11:25:33 +0800
Subject: [PATCH 0578/5242] MLK-11614 ARM: dts: imx6qp-sabresd: Enable
 prefetch resolve gasket(PRG) engines

commit  0b112c1ce19477d52ac5fec5f05695492fc6d200 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables the two prefetch resolve gasket(PRG) engines on the
imx6qp-sabresd platform.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    2 +-
 arch/arm/boot/dts/imx6qp-sabresd.dts   |   10 ++++++++++
 2 files changed, 11 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 430c6ed..e71e6f4 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -52,7 +52,7 @@
 		};
 	};
 
-	memory@10000000 {
+	memory: memory@10000000 {
 		reg = <0x10000000 0x40000000>;
 	};
 
diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 3038ccb..3864021 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -69,3 +69,13 @@
 &pre4 {
 	status = "okay";
 };
+
+&prg1 {
+	memory-region = <&memory>;
+	status = "okay";
+};
+
+&prg2 {
+	memory-region = <&memory>;
+	status = "okay";
+};
-- 
1.7.9.5

